test/game_team_test.rb

```diff
@@ -0,0 +1,81 @@
+require 'simplecov'
+SimpleCov.start
```

Comment:
not sure why my coverage istn higher

Reply:
I have basically the same thing (minus using fixture file) and my coverage is 100% so also not sure

Reply:
We'll keep an eye on it as we go but my theory is that not everything has a test written for it? It's probably something small :)